modules/UI/shared_video/SharedVideoThumb.js

```diff
@@ -0,0 +1,80 @@
+/* global $ */
+import SmallVideo from '../videolayout/SmallVideo';
+
+const logger = require("jitsi-meet-logger").getLogger(__filename);
+
+export default function SharedVideoThumb (url, videoType, VideoLayout)
+{
+    this.id = url;
+
+    this.url = url;
+    this.setVideoType(videoType);
+    this.videoSpanId = "sharedVideoContainer";
+    this.container = this.createContainer(this.videoSpanId);
+    this.$container = $(this.container);
+    this.container.onclick = this.videoClick.bind(this);
+    this.bindHoverHandler();
+    SmallVideo.call(this, VideoLayout);
+    this.isVideoMuted = true;
+}
+SharedVideoThumb.prototype = Object.create(SmallVideo.prototype);
+SharedVideoThumb.prototype.constructor = SharedVideoThumb;
+
+/**
+ * hide display name
+ */
+
+SharedVideoThumb.prototype.setDeviceAvailabilityIcons = function () {};
+
+SharedVideoThumb.prototype.avatarChanged = function () {};
+
+SharedVideoThumb.prototype.createContainer = function (spanId) {
+    var container = document.createElement('span');
+    container.id = spanId;
+    container.className = 'videocontainer';
+
+    // add the avatar
+    var avatar = document.createElement('img');
+    avatar.className = 'sharedVideoAvatar';
+    avatar.src = "https://img.youtube.com/vi/" + this.url + "/0.jpg";
+    container.appendChild(avatar);
+
+    const displayNameContainer = document.createElement('div');
+    displayNameContainer.className = 'displayNameContainer';
+    container.appendChild(displayNameContainer);
+
+    return container;
+};
+
+/**
+ * The thumb click handler.
+ */
+SharedVideoThumb.prototype.videoClick = function () {
+    this.VideoLayout.handleVideoThumbClicked(this.url);
+};
+
+/**
+ * Logs the removal of the SharedVideoThumb instance.
+ *
+ * @returns {void}
+ */
+SharedVideoThumb.prototype.remove = function () {
+    logger.log("Remove shared video thumb", this.id);
```

Comment:
If I follow this correctly this method will be called inside VideoLayout.removeParticipantContainer (line 840) and we're already logging the remove of the thumbnail there, so do we really need this?